CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
-## 0.14 (October 7, 2015)
+## 0.14.1 (October 28, 2015)
+
+### React DOM
+- Fixed bug where events wouldn't fire in old browsers when using React in development mode
+- Fixed bug preventing use of `dangerouslySetInnerHTML` with Closure Compiler Advanced Mode
```

Comment:
nit: lowercase "mode" is how https://developers.google.com/closure/compiler/docs/limitations?hl=en does it